packages/client/src/__tests__/integration/happy/logging/library.ts

```diff
@@ -104,34 +104,34 @@ test('interactive transactions logging - library', async () => {
   replaceTimeValues(onQuery)
 
   expect(onQuery.mock.calls).toMatchInlineSnapshot(`
-Array [
-  Array [
-    Object {
-      duration: 0,
-      params: [],
-      query: BEGIN,
-      target: quaint::connector::metrics,
-      timestamp: 1970-01-01T00:00:00.000Z,
-    },
-  ],
-  Array [
-    Object {
-      duration: 0,
-      params: [0],
-      query: SELECT "public"."User"."id" FROM "public"."User" WHERE 1=1 OFFSET $1,
-      target: quaint::connector::metrics,
-      timestamp: 1970-01-01T00:00:00.000Z,
-    },
-  ],
-  Array [
-    Object {
-      duration: 0,
-      params: [],
-      query: COMMIT,
-      target: quaint::connector::metrics,
-      timestamp: 1970-01-01T00:00:00.000Z,
-    },
-  ],
-]
-`)
+    Array [
+      Array [
+        Object {
+          duration: 0,
+          params: [],
+          query: BEGIN,
+          target: quaint::connector::metrics,
+          timestamp: 1970-01-01T00:00:00.000Z,
+        },
+      ],
+      Array [
+        Object {
+          duration: 0,
+          params: [0],
+          query: SELECT "public"."User"."id" FROM "public"."User" WHERE 1=1 OFFSET $1 /* traceparent=00-00-00-00 */,
```

Comment:
I think this needs to only appear if we are actually tracing, wdyt? Let's take a team decision for what we want to do.

Reply:
I think adding it in here is acceptable. The final `00` means this shouldn't be sampled. From my understanding of tracing it seems ok to pass a trace-id and span id to other components even if it is not being sampled.